README.md

```diff
@@ -54,7 +54,7 @@ LoggingSystem.bootstrap(StreamLogHandler.standardError)
 
 For further information, please check the [API documentation][api-docs].
 
-#### Selecting a logging backend implementation (applications only)
+#### Backends
```

Comment:
Mixed feelings about "just" renaming the heading. 

Note that here it's part of a narrative:

- Getting started
  - Adding the dependency
  - Let's log
  - Default Logger behavior
  - Selecting a logging backend implementation (applications only)

It's a narrative section / tutorial. Just changing it to have a better anchor does not feel right.

How about:

```markdown
#### Selecting a logging backend implementation (applications only)		

Usually you will want to configure the logging system with a specific backend of your choosing, depending on your use case. See the list or available backends below to pick a suitable backend for you.

## Available Backends
or 
## Logging Backends
or 
## Logger Backends
or 
## Backends

While the Logging API is still relatively young, a number of backend implementations are already available. If the backend you are interested in is not available yet and you'd like to build it, see the "Implementation considerations" section below explaining how to do so. 

List of existing SwiftLog API compatible libraries:
```

WDYT?

Reply:
Alternatively, you use inline HTML to provide a new anchor without changing the heading text:

```markdown
<a name="backends"></a>
#### Selecting a logging backend implementation
```

Reply:
I'm +1 to a hidden link. @weissi @ktoso what do you think?

Reply:
I think @mattt's anchor idea is the best, +1 from me

Reply:
Right that works, it's also what was initially starting out with when commenting but then backed out of it since it's harder to discover.

But yeah, as a minimal improvement I'd be ok with that.